portal/app/core/assets/css/dashboard.css

```diff
@@ -69,6 +69,10 @@
   font-weight: 600;
 }
 
+/*
+ * Loads
+ */
```

Comment:
rounded sliders is a part of loads.html, I wanted to distinguish the styles elements of each partial template inside dashboard.css instead of creating a new css file for every html file. The commenting does look a little confusing, do you have any suggestions to fix that?

Reply:
no that's fine